Silicon/MeteorlakePkg/Include/Library/PchInfoLib.h

```diff
@@ -738,4 +738,77 @@ BOOLEAN
 MtlSocIsPchAttached (
   VOID
 );
+
+/**
+  Get Serial IO SPI controller PCIe Device Number
+
+  @param[in]  SpiNumber       Serial IO SPI controller index
+
+  @retval Serial IO SPI controller PCIe Device Number
+**/
+UINT8
+MtlPchSerialIoSpiDevNumber (
```

Comment:
For the library API functions, often we will add EFIAPI in both header and c files.